.gitignore

```diff
@@ -14,4 +14,4 @@ hs_err_pid*
 .idea/
 *.iml
 .gradle/
-
+build/
```

Comment:
I didn't add the build folder itself - I only added a line to gitignore so that it would be ignored. Maybe we're talking at cross purposes ... Does it look to you like I added the folder? I can double check in the morning but there should only be two new files in this pull request and two files with small changes.